openedx/features/enterprise_support/api.py

```diff
@@ -233,6 +234,32 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS and getattr(settings, 'ENABLE_ENTERPRISE_INTEGRATION', True)
 
 
+def enterprise_customer_for_request(request, tpa_hint=None):
```

Comment:
This one is a tough call. The philosophical line I've been trying to toe in terms of "what goes in Enterprise; what goes in enterprise-support in edx-platform?" is a tough one - we want to provide good levels of abstraction in the enterprise support app so that we don't need to go all the way out to edx-enterprise elsewhere in the platform. So, I think there's an argument to be made, long-term, that some of the logic of this method should be moved over, but I remain convinced that this method should exist here.